jdaviz/configs/imviz/tests/test_parser.py

```diff
@@ -407,29 +407,29 @@ def test_parse_hst_drz(self, imviz_helper):
         assert_allclose(sky.dec.deg, 10.802045612042956, rtol=1e-3)
         data_unit = u.electron / u.s
         assert_quantity_allclose(tbl[0]['background'], 0.0014 * data_unit)
-        assert_quantity_allclose(tbl[0]['sum'], 126.789257 * data_unit, rtol=1e-3)
+        assert_quantity_allclose(tbl[0]['sum'], 115.944737 * data_unit, rtol=1e-3)
```

Comment:
Hmm. Sum decreased and the object is rounder. Maybe they removed an artifact in the update.